Doc/c-api/init.rst

```diff
@@ -1091,6 +1094,13 @@ with sub-interpreters:
    When the function returns, the current thread will hold the GIL and be able
    to call arbitrary Python code.  Failure is a fatal error.
 
+   If sub-interpreters are active, this function puts you in the global
+   interpreter (created by :c:func:`Py_Initialize`). If you create a sub-interpreter
+   right after you call this function, then the current thread is switched to that
```

Comment:
```suggestion
   right after you call this function, then the current thread is switched to that
```
By "current thread", you mean the thread's interpreter state?

Reply:
Yeah, I'll fix that.